test/unit/foreman/access_permissions_test.rb

```diff
@@ -1,6 +1,7 @@
 require 'test_helper'
 require 'foreman/access_control'
```

Comment:
Thanks for fixing, but please reply/leave a comment so the reviewer knows it's changed.